docs/dictionary.rst

```diff
@@ -0,0 +1,1101 @@
+*abbreviation for* Application Programmer Interface  
+
+	A software-based intermediary used to exchange data, often between two different platforms. Communication between different cloud platforms is mediated by various APIs, such as :ref:`dict TES`.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict AWS:
+
+AWS
+---
+*abbreviation for* Amazon Web Services  
+
+	A provider of cloud services, most notably cloud computing and cloud storage, available on-demand and hosted by Amazon. :ref:`dict Seven Bridges` is an example of a system that is powered by AWS, and can launch workflows on :ref:`dict EC2` instances.  
+
+see also :ref:`dict GCP`  
+
+Further reading: `<https://docs.aws.amazon.com/index.html?nc2=h_ql_doc_do>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict BD Catalyst:
+
+BD Catalyst
+-----------
+*abbreviation for* :ref:`dict BioData Catalyst`  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict BDC:
+
+BDC
+---
+pronounced "bee-dee-see"  
+
+*abbreviation for* :ref:`dict Biodata Catalyst`  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Biodata Catalyst:
+
+Biodata Catalyst
+----------------
+	An initiative funded by :ref:`dict NHLBI` to connect several cloud-based bioinformatics platforms together to increase reproducibility in bioinformatics. Involves Dockstore, :ref:`dict Terra`, Seven Bridges, Gen3, and PIC-SURE.  
+
+Further reading: `<https://biodatacatalyst.nhlbi.nih.gov/>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict Cancer Genomics Cloud:
+
+Cancer Genomics Cloud
+---------------------
+	A cloud platform by :ref:`dict Seven Bridges` and funded by :ref:`dict NCI` for bioinformatics analysis.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict categories:
+
+categories
+----------
+	A group of workflows or tools on Dockstore with a similar scientific purpose.  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict CGC:
+
+CGC
+---
+*abbreviation for* :ref:`dict Cancer Genomics Cloud`  
+
+
+.. updated 2022-05-16  
+
+
+
+.. _dict CLI:
+
+CLI
+---
+*abbreviation for* Command Line Interface  
+
+	A program that can be interacted with on the command line, usually via "Terminal" on MacOS and Linux or "cmd"/Command Prompt on Windows. CLI programs generally do not have a graphical user interface.  
+
+Further reading: `<https://en.wikipedia.org/wiki/Command-line_interface>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict cloud computing:
+
+cloud computing
+---------------
+	Doing computational tasks on a remote machine that is made available on-demand without the user having to manage all aspects of it. Generally implies that the user is essentially renting computational resources from someone else. Well-known cloud providers include :ref:`dict GCP`, :ref:`dict AWS`, Microsoft Azure, and Alibaba Cloud.  
+
+Further reading: `<https://en.wikipedia.org/wiki/Cloud_computing>`_  
+
+.. updated 2022-05-16  
+
+
+
+.. _dict collections:
+
+collections
+-----------
+	A group of workflows or tools on Dockstore associated with a particular :ref:`dict organization`.  
```

Comment:
But why is it associated with a particular organization (i.e. we should probably describe our stuff a little better)

Reply:
Agreed, which is why I was hoping to get some input on a few of these entries. I get that collections tend to be all for a particular purpose, but the difference between that and a category can get fuzzy - if the Broad organization has a collection for GATK workflows, the meaningful difference between that and a category of GATK workflows is just that the collection implies that those workflows are maintained by the Broad while the category does not imply that. But as soon as I say that in a definition, it implies that collections are always maintained by their organization, ie it puts a burden on organizations. How should I approach this?

Reply:
That's more thought than I had put into it. I was thinking something like "A collection is a group of Dockstore workflows that an organization found useful, created themselves, or found interesting. See their description to see why the organization collected workflows/entries in a collection"